Source/WebCore/platform/graphics/mac/controls/SliderTrackMac.mm

```diff
@@ -33,21 +33,22 @@
 
 namespace WebCore {
 
-SliderTrackMac::SliderTrackMac(SliderTrackPart& part, ControlFactoryMac& controlFactory)
+SliderTrackMac::SliderTrackMac(ControlPart& part, ControlFactoryMac& controlFactory)
     : ControlMac(part, controlFactory)
 {
+    ASSERT(part.type() == StyleAppearance::SliderHorizontal || part.type() == StyleAppearance::SliderVertical);
 }
 
 FloatRect SliderTrackMac::rectForBounds(const FloatRect& bounds, const ControlStyle& style) const
 {
     static constexpr int sliderTrackWidth = 5;
     float trackWidth = sliderTrackWidth * style.zoomFactor;
 
-    auto& sliderTrackPart = owningSliderTrackPart();
+    auto& sliderTrackAppearance = owningPart();
```

Comment:
Why did we change the name to `sliderTrackAppearance`? `owningPart()` returns a `ControlPart` not a `ControlAppearance`?

Reply:
Fixed